### ISSUE TITLE
Feature Request: Taalspecificatie voor responses in Vera API

### ISSUE THREAD

**Issue by renkosteenbeek** [Aedes-datastandaarden/vera-openapi#73]
## Huidige situatie
De Vera API standaard biedt momenteel geen mogelijkheid om de taal van de API-response te specificeren.

## Gewenste functionaliteit
Ik stel voor een mechanisme toe te voegen waarmee API-gebruikers de gewenste taal voor de response kunnen aangeven. Het gaat hier om de veld-waarden zoals "omschrijving", niet om de namen van de velden.

### Voorgestelde implementatie:
Ondersteuning voor het specificeren van de taal via de `Accept-Language` HTTP-header.

## Voordelen
Hiermee wordt de mogelijkheid geopend om meertaligheid aan te bieden. Het zou bijvoorbeeld kunnen dat een veld in het ERP systeem in diverse talen wordt vertaald; en het moet weergegeven in de taal van de huurder.

## Voorbeeld

Via HTTP-header:
   ```
   GET /api/resource
   Accept-Language: nl
   ```

**Comment by peterjan-aedesdatastandaarden** [comment]
Hoi Renco

Bedankt hiervoor. Om als client de voorkeurstaal te kunnen doorgeven is volgens ons een nuttige toevoeging. Voor een juiste verwachting (tussen client/server) moet dit in een 'best practice' uitgewerkt worden. HTTP kent een aantal algemene headers en request- en response headers. Als VERA willen we die niet opleggen/opnemen in de API-specificaties zelf. Wat we wel gaan doen 
 is het gebruik van headers beschrijven op de WIKI van VERA. Kan je je daar in vinden?

**Comment by renkosteenbeek** [comment]
Ja, ik snap wat je zegt. Het is een goed idee dit toe te voegen in de best practice inderdaad.
